### PR TITLE
[4.0] close collapse when opening a dropdown

### PR DESCRIPTION
**Pull request by hans2103**
Pull Request for Issue #28952 .

### Summary of Changes

This PR will add some javascript.
When you click on a dropdown element (data-bs-toggle="dropdown") , all open collapse elements (data-bs-toggle="collapse") will close

### Testing Instructions

- Joomla 4.4-dev with demo data
- resize viewport to mobile
- toggle any of the three buttons in the footer to open / close the element
- see the issue as shown in the first screencapture movie
- run `npm ci` to compile both css and js
- toggle any of the three buttons in the footer to open / close the element
- see the issue as shown in the second screencapture movie

### Actual result BEFORE applying this Pull Request

https://github.com/user-attachments/assets/7a45ec99-4f5a-42a7-8566-28d03ba597e2

### Expected result AFTER applying this Pull Request

https://github.com/user-attachments/assets/d605acff-d038-4c2f-bb4c-6bf7e9b26447

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [ ] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [ ] No documentation changes for manual.joomla.org needed

### CODE REVIEW

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 3d7b7dfab9477afc7d5dd187d041679e1fd88645<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/43891">issues.joomla.org/tracker/joomla-cms/43891</a>.</sub>

**Comment by richard67** [comment]
@hans2103 Please fix the javascript cs issues reported here https://ci.joomla.org/joomla/joomla-cms/77860/1/33 :

```
/drone/src/build/media_source/templates/administrator/atum/js/template.es6.js
  251:25  error  Expected parentheses around arrow function argument  arrow-parens
  252:36  error  Unexpected unnamed function                          func-names
  252:36  error  Unexpected function expression                       prefer-arrow-callback
  253:29  error  Expected parentheses around arrow function argument  arrow-parens
  256:22  error  Missing trailing comma                               comma-dangle
```

**Comment by hans2103** [comment]
@richard67 lint:js issue have been fixed
@brianteeman please feel free to test again. (same goes for the rest of the world, but you are faster)

File: build/media_source/templates/administrator/atum/js/template.es6.js [hunk 1]
```diff
@@ -242,3 +242,20 @@ window.addEventListener('joomla:menu-toggle', (event) => {
     changeLogo(event.detail);
   }
 });
+
+/**
+ * Close any open data-bs-toggle="collapse" when opening a data-bs-toggle="dropdown"
+ *
+ * since 4.4
```
Comment (brianteeman) [inline build/media_source/templates/administrator/atum/js/template.es6.js]:
```suggestion
 * @since 4.4
```

**Comment by exlemor** [comment]
I have tested this item :white_check_mark: successfully on c3a925f0b0c6fc452c3effe84da1c5fcb8d48f16<br /><br />I have tested this successfully... :)

( Thanks for this fix, I sometimes have to use my mobile phone and this will help! )<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/43891">issues.joomla.org/tracker/joomla-cms/43891</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/43891">issues.joomla.org/tracker/joomla-cms/43891</a>.</sub>

**Comment by laoneo** [comment]
Thanks!

**Comment by Radek-Suski** [comment]
Given that accordions are also using data-bs-toggle="collapse" it also causes all accordions tabs to close.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/43891">issues.joomla.org/tracker/joomla-cms/43891</a>.</sub>

**Pull request merged**